hs_geographic_feature_resource/page_processors.py

```diff
@@ -103,6 +103,7 @@ def landing_page(request, page):
         context['ori_coverage_form'] = ori_coverage_form
         context['geom_information_form'] = geom_information_form
 
+    context['edit_mode'] = edit_resource
```

Comment:
@pkdash the frontend is expecting to read this value to determine current mode: View or Edit.
If it is Edit mode, the frontend will show a custom warning popup if users try to delete a mandatory file from the Feature Res